### ISSUE TITLE
Thermostat

### ISSUE THREAD

**Issue by sealedge** [kuestess/homebridge-platform-insteonlocal#201]
I have everything working through HomeBridge -> Google Home... however, I cannot see the thermostats on Google Home.

Here is my config file where thermostat is mentioned. Please advise.
                },
                {
                    "name": "Thermo1",
                    "category": 5,
                    "subcategory": 11,
                    "sku": "2441TH ",
                    "deviceID": "345CF6",
                    "dimmable": "false",
                    "deviceType": "thermostat"
                },
                {
                    "name": "Thermo2",
                    "category": 5,
                    "subcategory": 11,
                    "sku": "2441TH ",
                    "deviceID": "3459E5",
                    "dimmable": "false",
                    "deviceType": "thermostat"
                },
                {
                    "name": "Thermo3",
                    "category": 5,
                    "subcategory": 11,
                    "sku": "2441TH ",
                    "deviceID": "345B86",
                    "deviceType": "thermostat",
                    "dimmable": "false"
                }

**Comment by Coasterfreak212** [comment]
I don't think this Plugin supports Thermostats. It's not listed as one of the controllable things in the Readme.

I have 2 Insteon Thermostats as well and I'd like to have thermostat support as well.

**Comment by kuestess** [comment]
@sealedge @Coasterfreak212 No thermostat support currently, but I can work on getting it added in an update.

**Comment by Coasterfreak212** [comment]
> @sealedge @Coasterfreak212 No thermostat support currently, but I can work on getting it added in an update.

Awesome, thank you @kuestess ! Much appreciated!

**Comment by sealedge** [comment]
Yes thank you ! @kuestess. With thermostat, the Insteon disappearing act will be resolved for now.

**Comment by funkyvisions** [comment]
> @sealedge @Coasterfreak212 No thermostat support currently, but I can work on getting it added in an update.

Thermostat support would be awesome now that Insteon has left us high and dry.

**Comment by sealedge** [comment]
For sure. Any update on this thermostat coding?

Jacques HabraTHIS MESSAGE IS CONFIDENTIAL. This e-mail message and any
attachments are proprietary and confidential information intended only for
the use of the recipient (s) named above. If you are not the intended
recipient, you may not print, distribute, or copy this message or any
attachments.   If you have received this communication in error, please
notify the sender by return e-mail and delete this message and any
attachments from your computer.
